app/models/user.py

```diff
@@ -53,9 +77,19 @@ class User(UserMixin, db.Model):
     email = db.Column(db.String(64), unique=True, index=True)
     password_hash = db.Column(db.String(128))
     role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))
+
+    donor_level_id = db.Column(db.Integer, db.ForeignKey('donorlevels.id'))
```

Comment:
if we are changing name of table, should change to ForeignKey('donor_levels.id')